buildbot/configure.py

```diff
@@ -71,25 +75,27 @@ def do_configure(args):
         "-DLLVM_ENABLE_DOXYGEN={}".format(llvm_enable_doxygen),
         "-DLLVM_ENABLE_SPHINX={}".format(llvm_enable_sphinx),
         "-DBUILD_SHARED_LIBS={}".format(llvm_build_shared_libs),
-        "-DSYCL_ENABLE_XPTI_TRACING=ON", # Explicitly turn on XPTI tracing
-        "{}".format(args.cmake_opts),
-        llvm_dir
-    ]
+        "-DSYCL_ENABLE_XPTI_TRACING=ON" # Explicitly turn on XPTI tracing
+    ] + args.cmake_opt
+
 
     if not args.no_ocl:
       cmake_cmd.extend([
             "-DOpenCL_INCLUDE_DIR={}".format(ocl_header_dir),
             "-DOpenCL_LIBRARY={}".format(icd_loader_lib)])
 
+    # Add path to root CMakeLists.txt
+    cmake_cmd.append(llvm_dir)
```

Comment:
I recently figured out that path to CMakeLists.txt dir can be anywhere on the command line (i.e. you don't have to remove it from line 76). I still like it as a separate statement, improves readability.